flight/PiOS/inc/pios_servo.h

```diff
@@ -33,8 +33,18 @@
 
 #define PIOS_SERVO_MAX_BANKS 6
 
+/* Used in out_rate.  If 65535, we really mean 65535Hz.  Values close to that
+ * are reserved/abused for dshot support.  0 is used for sync pwm per legacy
+ */
+enum pios_servo_shot_type {
+	SHOT_ONESHOT = 0,
```

Comment:
Yes, but shouldn't we abandon it?  It's a support issue for no reason to tell everyone to use "synchronous PWM" when they want "oneshot".